<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_One</class>
 <widget class="QWidget" name="Window_One">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>340</y>
     <width>560</width>
     <height>485</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1580</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 4px;
    border-radius: 10px;
    border-color: black;
    min-width: 10em;
	background-color: Azure;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>155</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_point">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #400261</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_rb">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_lb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_x">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #400261</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_plus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mult">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5E048D</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_pow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4A036F</string>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_minus">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_mod">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>660</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #400261</string>
   </property>
   <property name="text">
    <string>mod</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_div">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #54037E</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cos">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5E048D</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tan">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #54037E</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_asin">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>asin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_atan">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #54037E</string>
   </property>
   <property name="text">
    <string>atan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_acos">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5E048D</string>
   </property>
   <property name="text">
    <string>acos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sqrt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4A036F</string>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_log">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>580</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4A036F</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ln">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>660</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #400261</string>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_eq">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>85</width>
     <height>235</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #B56200</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>85</width>
     <height>155</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6F07A6</string>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_x">
   <property name="geometry">
    <rect>
     <x>1125</x>
     <y>140</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	background-color: Azure;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_eq">
   <property name="geometry">
    <rect>
     <x>1045</x>
     <y>120</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>X = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_in">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>200</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>X ∈[</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points_2">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>195</y>
     <width>20</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rb_2">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>200</y>
     <width>20</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_x_from">
   <property name="geometry">
    <rect>
     <x>1125</x>
     <y>210</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	background-color: Azure;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_x_to">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>210</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	background-color: Azure;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rb">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>280</y>
     <width>20</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_y_to">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>290</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	background-color: Azure;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_in">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>280</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>Y ∈[</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_y_from">
   <property name="geometry">
    <rect>
     <x>1125</x>
     <y>290</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	background-color: Azure;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>275</y>
     <width>20</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nasalization</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: purple</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/calculator_background.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>plot</zorder>
  <zorder>Result</zorder>
  <zorder>button_1</zorder>
  <zorder>button_4</zorder>
  <zorder>button_2</zorder>
  <zorder>button_3</zorder>
  <zorder>button_6</zorder>
  <zorder>button_5</zorder>
  <zorder>button_7</zorder>
  <zorder>button_8</zorder>
  <zorder>button_9</zorder>
  <zorder>button_0</zorder>
  <zorder>button_point</zorder>
  <zorder>button_rb</zorder>
  <zorder>button_lb</zorder>
  <zorder>button_x</zorder>
  <zorder>button_plus</zorder>
  <zorder>button_mult</zorder>
  <zorder>button_pow</zorder>
  <zorder>button_minus</zorder>
  <zorder>button_mod</zorder>
  <zorder>button_div</zorder>
  <zorder>button_sin</zorder>
  <zorder>button_cos</zorder>
  <zorder>button_tan</zorder>
  <zorder>button_asin</zorder>
  <zorder>button_atan</zorder>
  <zorder>button_acos</zorder>
  <zorder>button_sqrt</zorder>
  <zorder>button_log</zorder>
  <zorder>button_ln</zorder>
  <zorder>button_eq</zorder>
  <zorder>button_clear</zorder>
  <zorder>line_x</zorder>
  <zorder>label_x_eq</zorder>
  <zorder>label_x_in</zorder>
  <zorder>label_points_2</zorder>
  <zorder>label_rb_2</zorder>
  <zorder>line_x_from</zorder>
  <zorder>line_x_to</zorder>
  <zorder>label_rb</zorder>
  <zorder>line_y_to</zorder>
  <zorder>label_y_in</zorder>
  <zorder>line_y_from</zorder>
  <zorder>label_points</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
